github/ipld.yml

```diff
@@ -3226,108 +3146,41 @@ teams:
       maintainer:
         - rvagg
         - SgtPooki
-      member:
-        - hacdias
-        - lidel
-        - olizilla
-        - whizzzkid
-    privacy: closed
-  Java Team:
```

Comment:
I think we can get rid of this, but we need to maintain ianopolous permission in the one repo that uses this team.